src/semantic-router/pkg/metrics/metrics.go

```diff
@@ -102,6 +102,15 @@ var (
 		[]string{"model"},
 	)
 
+	// ModelCostUSD tracks the total USD cost attributed to each model
+	ModelCostUSD = promauto.NewCounterVec(
```

Comment:
should USD be a specific naming suffix for this metric?

Reply:
For the cons, maybe we should have clear docs around metrics, it is a bit bad to add more metrics when units increase linearly

Reply:
adding a currency label is a good idea!

Reply:
Should I include this change in the follow-up PR or in this one?

Reply:
I created an issue for tracking.
https://github.com/vllm-project/semantic-router/issues/75